### PR TITLE
Add note about `uvx` to `uv tool run` short help

### PR DESCRIPTION
**Pull request by JacobCoffee**
## Summary

- Adds detail to the `uv tool run --help` CLI  that lets users know about the shorthand, `uvx`

## Test Plan

Building and running CLI
```
…📝✓] via 🐋 orbstack via 🎁 v0.4.16 via  pyenv via ⚙️ v1.81.0on ☁️  (us-east-2) took 3s 
➜ ./target/debug/uv tool run --help
Run a command provided by a Python package. Also available via the alias `uvx`.

Usage: uv tool run [OPTIONS] [COMMAND]

...

You can also use `uvx` as an alias for `uv tool run`. 
Use `uv help tool run` for more details.
```

### CODE REVIEW

File: crates/uv-cli/src/lib.rs [hunk 1]
```diff
@@ -3174,6 +3174,11 @@ pub enum ToolCommand {
     ///
     /// Packages are installed into an ephemeral virtual environment in the uv
     /// cache directory.
+    #[command(
+        about = "Run a command provided by a Python package. Also available via the alias `uvx`.",
+        after_help = "You can also use `uvx` as an alias for `uv tool run`. \n\
```
Comment (zanieb) [inline crates/uv-cli/src/lib.rs:3179]: Oh there's already a comment for this above in the long help i.e. `uv help tool run`. I'm a bit hesitant to add it to the short help?

Comment (JacobCoffee) [reply crates/uv-cli/src/lib.rs:3179]: Sorry! Do you mean `Run a command provided by a Python package` or something else entirely? 😬

Comment (JacobCoffee) [reply crates/uv-cli/src/lib.rs]: Hoping i understood that you meant the former: [73d1b39](https://github.com/astral-sh/uv/pull/7695/commits/73d1b393f9b54e46446206afdf73ecbfefa97fbb)
```
➜ ./target/debug/uv tool run --help | head
Also available via the alias `uvx`.

Usage: uv tool run [OPTIONS] [COMMAND]

...
```

Comment (zanieb) [reply crates/uv-cli/src/lib.rs]: `uv help tool run` already includes 

> `uvx` is provided as a convenient alias for `uv tool run`, their behavior is identical.

So we'd have a duplicate message here which feels weird.

Comment (zanieb) [reply crates/uv-cli/src/lib.rs]: I think if anything we'd use just `after_help` and not modify the `about`? You'd need to set `after_long_help` to an empty string.

Comment (JacobCoffee) [reply crates/uv-cli/src/lib.rs]: Maybe [882b4d3](https://github.com/astral-sh/uv/pull/7695/commits/882b4d395d1d905fa19c5851f7b46260605217d8)?

**Review by zanieb — APPROVED** [review]
Thanks! I futzed with this a bit, not in love but can't find anything better.

**Pull request merged**